byteverse_24/src/app/components/KnifeImage.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from "react";
+import { motion } from "framer-motion";
+import Image from "next/image";
+
+const KnifeImage = () => {
+  const [animationState1, setAnimationState1] = useState("initial");
+  const [animationState2, setAnimationState2] = useState("initial");
+
+  const variants1 = {
+    initial: {
+      opacity: 1,
+      x: 0,
+      y: 0,
+    },
+    moveLeft: {
+      opacity: 1,
+      x: "-10vw",
+      y: "-10vh",
+      transition: {
+        duration: 0.6,
+        ease: "easeOut",
+        delay: 1,
+      },
+    },
+
+    center: {
+      opacity: 1,
+      x: 0,
+      y: 0,
+      transition: {
+        duration: 0.1,
+        ease: "easeIn",
+      },
+    },
+  };
+  const variants2 = {
+    initial: {
+      opacity: 1,
+      x: 0,
+      y: 0,
+    },
+    moveLeft: {
+      opacity: 1,
+      x: "10vw",
+      y: "-10vh",
+      transition: {
+        duration: 0.6,
+        ease: "easeOut",
+
+        delay: 1,
+      },
+    },
+
+    center: {
+      opacity: 1,
+      x: 0,
+      y: 0,
+      transition: {
+        duration: 0.1,
+        ease: "easeIn",
+      },
+    },
+  };
+
+  const handleAnimationComplete1 = () => {
+    if (animationState1 === "initial") {
+      setAnimationState1("moveLeft");
+    } else if (animationState1 === "moveLeft") {
+      setAnimationState1("center");
+    } else {
+      setAnimationState1("initial");
+    }
+    console.log("Animation 1 complete. Current state:", animationState1);
```

Comment:
@dev04sa 
Always remember, when pushing your code onto production branch, you must remove any logs to the console.